### ISSUE TITLE
Entry and exit delay not synchronized - Alarmo Keypad v2

### ISSUE THREAD

**Issue by Eyco1981** [ImSorryButWho/HomeAssistantNotes#12]
With Alarmo version 1.9.0 the following breaking change has been made:
```
This release removes the expiration and delay property from the alarm entities. They were previously used to communicate exit/entry delays with alarmo-card. This is now handled via websockets.
```
Due to that the entry and exit delays aren't synchronized with the Keypad. 
As the attribute "delay" has been removed, the value "{{ state_attr(alarm, 'delay') }}" in the sequences for "alarm_arming" and "alarm_pending" doesn't work anymore.

**Comment by ImSorryButWho** [comment]
I'll talk to the Alarmo developers and see if they're willing to undo that change.  In the mean time, I have a fork which should work: https://github.com/ImSorryButWho/alarmo

**Comment by ImSorryButWho** [comment]
Fixed in Alarmo 1.9.1.
